unit_testing/cli/test_cli_sct_label_vertebrae.py

```diff
@@ -20,9 +23,34 @@ def test_sct_label_vertebrae_initz_error():
         sct_label_vertebrae.main(command.split())
 
 
-def test_sct_label_vertebrae_high_value_warning(caplog):
-    command = '-i sct_testing_data/t2/t2.nii.gz -s sct_testing_data/t2/t2_seg-manual.nii.gz -c t2 -initz 40,19'
+def test_sct_label_vertebrae_high_value_warning(caplog, tmp_path):
+    command = '-i sct_testing_data/t2/t2.nii.gz -s sct_testing_data/t2/t2_seg-manual.nii.gz -c t2 -initz 40,19 -ofolder ' + str(tmp_path)
     sct_label_vertebrae.main(command.split())
     assert 'Disc value not included in template.' in caplog.text
 
 
+def test_sct_label_vertebrae_clean_labels(tmp_path):
+    command = '-i sct_testing_data/t2/t2.nii.gz -s sct_testing_data/t2/t2_seg-manual.nii.gz -c t2 -initz 40,19 -clean-labels 1 -ofolder ' + str(os.path.join(str(tmp_path), 'clean'))
+    sct_label_vertebrae.main(command.split())
+    command = '-i sct_testing_data/t2/t2.nii.gz -s sct_testing_data/t2/t2_seg-manual.nii.gz -c t2 -initz 40,19 -ofolder ' + str(os.path.join(str(tmp_path), 'no_clean'))
```

Comment:
how come this one doesn't get a high value warning (see test_sct_label_vertebrae_high_value_warning)

Reply:
There is a warning in the log, but pytest does not print it ( I guess because it is in the API).

Reply:
my point was more: what is the reasoning for inputting "-initz 40,19", if this value creates a warning (see previous test)-- the point here is not to test the presence of a warning, but to verify the integrity of the output in "normal" (not with missing label) behaviour

Reply:
Ah yes right, I forgot to remove it.. It's been corrected.